portsyncd/linksync.cpp

```diff
@@ -215,7 +215,7 @@ void LinkSync::onMsg(int nlmsg_type, struct nl_object *obj)
     /* If netlink for this port has master, we ignore that for now
      * This could be the case where the port was removed from VLAN bridge
      */
-    if (master)
+    if (master && nlmsg_type == RTM_DELLINK)
```

Comment:
Let change the comments session above to state that we ignore the DELLINK message if port has master, this is applicable to the case where port was part or VLAN or LAG etc.   You should rename the PR title too.